src/DBUtils.ts

```diff
@@ -1,142 +1,64 @@
 
-    const user = await User.findOne({ where: { userid: userID } });
-    if (user) {
-        console.log(`user ${user.userid} found in Users`);
-        return true;
-    } else {
-        console.log(`user ${userID} NOT found in Users`);
-        return false;
-    }
-}
 
 /**
- * Adds a row in the "User" table, which contains a users discord ID and discord Username.
+ * Takes a rarity, randomly picks one gacha among that rarity, returns it
  * 
- * @param {string} userID - Users Discord ID
- * @param {string} username - Users Discord Username(not displayname)
+ * @param {string} rarity - Which rarity it should choose from
+ * @returns {Promise<Gacha>} - random 'rarity' gacha
  */
-export async function addUser(userID: string, username: string): Promise<void> {
+export async function getRandomGacha(rarity: string): Promise<Gacha> {
 
-    await User.create({ userid: userID, username: username });
-    console.log(`added ${userID} to Users`);
-}
+    const allGachaOfRarity = await Gacha.findAll({ where: { rarity: rarity } });
 
-/**
- * Searches for the 'balance' associated the given users Discord ID.
- * 
- * If no balance was found, it calls another function to create a new row in "Wallet"-
- * -which contains the users Discord ID, and 'balance' defaults to 0.
- * 
- * (balance is used in a monetary context $$$)
- * 
- * @param {string} userID - Users Discord ID
- * @returns {Promise<number>} -  Returns the users current balance in "Wallet" table.
- *                               Returns 0 if no number was found (default value).
- */
-export async function checkBalance(userID: string): Promise<number> {
-    const userBalance = await Wallet.findOne({ where: { userid: userID } });
-    if (userBalance) {
-        console.log(`${userID} has a balance of ${userBalance.balance}`);
-        return userBalance.balance;
-    } else {
-        await findOrAddUserWallet(userID);
-        return 0;
-    }
-}
+    const randomGacha = Math.floor(Math.random() * Math.floor(allGachaOfRarity.length));
 
-/**
- * Add or Subtract the 'balance' associated with the users Discord ID-
- * -in the "Wallet" table by 'amount'.
- * 
- * @param userID - Users Discord ID
- * @param amount - Positive or Negative integer to Add or Subtract 'balance' by
- */
-export async function addOrSubtractBalance(userID: string, amount: number): Promise<void> {
-    await Wallet.increment({ balance: amount }, { where: { userid: userID } });
-    console.log(`${userID}'s wallet has been changed by: ${amount}`);
-}
+    const gacha = allGachaOfRarity[randomGacha];
 
-/**
- * Checks if there is a row in the "Wallet" table that contains the users Discord ID-
- * -creates one if it is not found.
- * 
- * @param {string} userID - Users Discord ID
- */
-export async function findOrAddUserWallet(userID: string): Promise<void> {
-    await Wallet.findOrCreate({ where: { userid: userID } });
-    console.log(`Found or created Wallet for ${userID}`);
+    return gacha;
```

Comment:
Just return `allGachaOfRarity[randomGacha]` directly (same applies for functions below as well)